reactor-netty-http/src/main/java/reactor/netty/http/server/MicrometerHttpServerMetricsRecorder.java

```diff
@@ -34,12 +34,16 @@
  * @author Violeta Georgieva
  * @since 0.9
  */
-final class MicrometerHttpServerMetricsRecorder extends MicrometerHttpMetricsRecorder implements HttpServerMetricsRecorder {
+public class MicrometerHttpServerMetricsRecorder extends MicrometerHttpMetricsRecorder implements HttpServerMetricsRecorder {
```

Comment:
Let's keep the class final and make it just public with the corresponding constructor. Wdyt?